# Deep Learning for peptides

ğŸ”¬ Comprehensive List of Research Papers on Peptides and Deep Learning

âœ… __What sets us apart from similar resources:__

1. Versatile Tags: Organize and filter papers easily.
2. Easy Navigation: Internal links for quick jumps between sections and papers.
3. Expert Insights: Links to expert reviews and analysis.
4. [CSV Downloads](data/paper.csv): Quick access to paper data in `CSV` format.
5. Automation: Use [Liquid](https://liquid.readthedocs.io/en/latest/) templates to generate Markdown from `CSV`, making it easy to build your own paper repository. >>> [[Details](CONTRIBUTING.md)]

ğŸ“… _Papers last month, updated on 2024-10-14:_

**Beware of extreme calculated lipophilicity when designing cyclic peptides**  
Vasanthanathan Poongavanam, Duc Duy Vo & Jan Kihlberg  
[**2024**-9-19] >> [Nat. Chem. Biol.](https://doi.org/10.1038/s41589-024-01715-0) â€¢ [SI](https://www.nature.com/articles/s41589-024-01715-0#MOESM1) â€¢ [Weixin](https://mp.weixin.qq.com/s/B65rJB1i_xrP8fTfbQ3Taw) â€¢ cyclic/clogP

**PepINVENT: Generative peptide design beyond the natural amino acids**  
GÃ¶kÃ§e Geylan, Jon Paul Janet, Alessandro Tibo, Jiazhen He, Atanas Patronov, Mikhail Kabeshov, Florian David, Werngard Czechtizky, Ola Engkvist, Leonardo De Maria  
[**2024**-9-21] >> [Axive](https://doi.org/10.48550/arXiv.2409.14040) â€¢ RL/Molecular AI/AstraZeneca

**Design of linear and cyclic peptide binders of different lengths from protein sequence information**  
Qiuzhen Li, Efstathios Nikolaos Vlachos, Patrick Bryant  
[**2024**-10-12] >> [Arxive](https://doi.org/10.1101/2024.06.20.599739) â€¢ [zenodo](https://zenodo.org/uploads/11543503) â€¢ [EvoBind](https://github.com/patrickbryant1/EvoBind) â€¢ cyclic

**BindCraft: one-shot design of functional protein binders**  
Martin Pacesa, Lennart Nickel, ..., Sergey Ovchinnikov, Bruno E. Correia  
[**2024**-10-1] >> [Arxive](https://doi.org/10.1101/2024.09.30.615802) â€¢ high â€¢ [GitHub](https://github.com/martinpacesa/BindCraft)
  
<details>
<summary>ğŸ” Abstract</summary>
<p>BindCraft is an open-source, automated pipeline for <em>de novo</em> protein binder design, achieving experimental success rates of 10-100%. Using deep learning models like AlphaFold2, BindCraft generates high-affinity binders without the need for high-throughput screening or prior knowledge of binding sites. It has been successfully applied to challenging targets, including cell-surface receptors, allergens, and CRISPR-Cas9. In one example, the binders reduced IgE binding to birch allergens in patient samples, showcasing its potential in therapeutics, diagnostics, and biotechnology.</p>
</details>  

ğŸ“Œ _Papers pined:_

**PepINVENT: Generative peptide design beyond the natural amino acids**  
GÃ¶kÃ§e Geylan, Jon Paul Janet, Alessandro Tibo, Jiazhen He, Atanas Patronov, Mikhail Kabeshov, Florian David, Werngard Czechtizky, Ola Engkvist, Leonardo De Maria  
[**2024**-9-21] >> [Axive](https://doi.org/10.48550/arXiv.2409.14040) â€¢ RL/Molecular AI/AstraZeneca

**Full-Atom Peptide Design with Geometric Latent Diffusion**  
Xiangzhe Kong, Yinjun Jia, Wenbing Huang, Yang Liu  
[**2024**-2-21] >> NeurIPS/[Arxive](https://doi.org/10.48550/arXiv.2402.13555) â€¢ [code](https://github.com/THUNLP-MT/PepGLAD) â€¢ Full-Atom/Diffusion

**Target-Specific De Novo Peptide Binder Design with DiffPepBuilder**  
Fanhao Wang, Yuzhe Wang, Laiyi Feng, Changsheng Zhang, and Luhua Lai  
[**2024**-9-4] >> [JCIM](https://doi.org/10.1021/acs.jcim.4c00975) â€¢ high â€¢ [GitHub](https://github.com/YuzheWangPKU/DiffPepBuilder) â€¢ Diffusion/Luhua Lai/AfDeisgn/ProteinMPNN/MD
  
<details>
<summary>ğŸ” Abstract</summary>
<p>Despite the exciting progress in target-specific de novo protein binder design, peptide binder design remains challenging due to the flexibility of peptide structures and the scarcity of protein-peptide complex structure data. In this study, we curated a large synthetic data set, referred to as PepPC-F, from the abundant proteinâˆ’protein interface data and developed DiffPepBuilder, a de novo target-specific peptide binder generation method that utilizes an SE(3)-equivariant diffusion model trained on PepPC-F to codesign peptide sequences and structures. DiffPepBuilder also introduces disulfide bonds to stabilize the generated peptide structures. We tested DiffPepBuilder on 30 experimentally verified strong peptide binders with available proteinâˆ’peptide complex structures. DiffPepBuilder was able to effectively recall the native structures and sequences of the peptide ligands and to generate novel peptide binders with improved binding free energy. We subsequently conducted de novo generation case studies on three targets. In both the regeneration test and case studies, DiffPepBuilder outperformed AfDesign and RFdiffusion coupled with ProteinMPNN, in terms of sequence and structure recall, interface quality, and structural diversity. Molecular dynamics simulations confirmed that the introduction of disulfide bonds enhanced the structural rigidity and binding performance of the generated peptides. As a general peptide binder de novo design tool, DiffPepBuilder can be used to design peptide binders for given protein targets with three-dimensional and binding site information.</p>
</details>  

**Denovo design of modular peptide-binding proteins by superhelical matching**  
Kejia Wu, Hua Bai, ..., Emmanuel Derivery, Daniel Adriano Silva, David Baker  
[**2023**-3-5] >> [Nature](https://doi.org/10.1038/s41586-023-05909-9) â€¢ high â€¢ [data](https://files.ipd.uw.edu/pub/2023_modular_peptide_binding_proteins/all_data_modular_peptide_binding_proteins.tar.gz) â€¢ [GitHub](https://github.com/tjs23/prot_pep_scan) â€¢ David Baker
  
<details>
<summary>ğŸ” Abstract</summary>
<p>General approaches for designing sequence-specific peptide-binding proteins would have wide utility in proteomics and synthetic biology. However, designing peptide-binding proteins is challenging, as most peptides do not have defined structures in isolation, and hydrogen bonds must be made to the buried polar groups in the peptide backbone1â€“3. Here, inspired by natural and re-engineered proteinpeptide systems4â€“11, we set out to design proteins made out of repeating units that bind peptides with repeating sequences, with a one-to-one correspondence between the repeat units of the protein and those of the peptide. We use geometric hashing to identify protein backbones and peptide-docking arrangements that are compatible with bidentate hydrogen bonds between the side chains of the protein and the peptide backbone12. The remainder of the protein sequence is then optimized for folding and peptide binding. We design repeat proteins to bind to six different tripeptide-repeat sequences in polyproline II conformations. The proteins are hyperstable and bind to four to six tandem repeats of their tripeptide targets with nanomolar to picomolar affinities in vitro and in living cells. Crystal structures reveal repeating interactions between protein and peptide interactions as designed, including ladders of hydrogen bonds from protein side chains to peptide backbones. By redesigning the binding interfaces of individual repeat units, specificity can be achieved for non-repeating peptide sequences and for disordered regions of native proteins.</p>
</details>  

---

<p align="center">
  <a href="https://doi.org/10.1038/s41586-023-05909-9">
  <img src="cover.png" alt="deep learning for peptides">
  </a>
</p>

<p align='center'>
  <strong><a href='#0-benchmarks-and-datasets'>0) Benchmarks and Datasets</a></strong>
  <br>
  <a href="#01-benchmarks">Benchmarks</a> â€¢
  <a href="#02-datasets">Datasets</a> â€¢
  <a href="#03-similar-list">Similar List</a> â€¢
  <a href="#04-tools">Tools</a>
  <br>
  <strong><a href='#1-reviews'>1) Reviews</a></strong>
  <br><a href='#11-design'>Design</a> â€¢ 
  <a href='#12-interaction'>Interaction</a> â€¢ 
  <a href='#13-structure'>Structure</a>
  <br>
  <strong><a href='#2-representation'>2) Representation</a></strong>
  <br>
  <strong><a href='#3-proprty-prediction'>3) Proprty Prediction</a></strong>
  <br><a href='#31-traditional'>Traditional</a>
  <br>
  <strong><a href='#4-structure-modeling'>4) Structure Modeling</a></strong>
  <br>
  <strong><a href='#5-design'>5) Design</a></strong>
  <br><a href='#51-sequence-based'>Sequence-based</a> â€¢ 
  <a href='#52-structure-based'>Structure-based</a>
  <br>
  <strong><a href='#6-others'>6) Others</a></strong>
  <br><a href='#61-protac'>PROTAC</a> â€¢ 
  <a href='#62-protein-binders'>Protein Binders</a> â€¢ 
  <a href='#63-therapeutic'>Therapeutic</a>
  <br>
</p>

---

## 0. Benchmarks and Datasets

### 0.1 Benchmarks

#### 0.1.1 Sequence Benchmarks

#### 0.1.2 Structure Benchmarks

**Benchmarking AlphaFold2 on peptide structure prediction**  
Eli Fritz McDonald, Taylor Jones, Lars Plate, Jens Meiler, Alican Gulsevin  
[**2024**-1-5] >> [Structure](https://doi.org/10.1016/j.str.2022.11.012) â€¢ [SI](https://doi.org/10.1016/j.str.2022.11.012) â€¢ [Weixin](https://mp.weixin.qq.com/s/9mpyZXITVC6RBbNQmjJLcg) â€¢ AF

**Predicting Proteinâˆ’Peptide Interactions: Benchmarking Deep Learning Techniques and a Comparison with Focused Docking**  
Sudhanshu Shanker and Michel F. Sanner  
[**2024**-5-11] >> [JCIM](https://doi.org/10.1021/acs.jcim.3c00602) â€¢ [GitHub](https://github.com/sannerlab/benchmarking_2023) â€¢ Fold

#### 0.1.3 Evaluations

### 0.2 Datasets

### 0.2.1 Public Datasets

> A list of suggested peptide datasets

| Datasets | Description                                                                                                                                                                                                                                                         |
| -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| PDB      | The Protein Data Bank (PDB) is a database of 3D structural data of large biological molecules, such as proteins and nucleic acids. These data are gathered using experimental methods such as X-ray crystallography, NMR spectroscopy, or cryo-electron microscopy. |

#### 0.2.1 Sequence Datasets

#### 0.2.2 Structure Datasets

### 0.3 Similar List

> Some similar GitHub lists that include papers about peptide using deep learning

1. a
2. b

### 0.4 Guides

> Guides/Tutorials for beginners on GitHub

1. a
2. b

### 0.5 Tools

> Tools for peptide develpements

1. a
2. b

## 1. Reviews

### 1.1 Design

**Unlocking novel therapies: cyclic peptide design for amyloidogenic targets through synergies of experiments, simulations, and machine learning**  
Daria de Raffele and Ioana M. Ilie  
[**2023**-11-7] >> [Chem. Commun.](https://doi.org/10.1039/D3CC04630C) â€¢ Cyclic


### 1.2 Interaction

**A comprehensive review of protein-centric predictors for biomolecular interactions: from proteins to nucleic acids and beyond**  
Pengzhen Jia, Fuhao Zhang, Chaojin Wu and Min Li  
[**2024**-3-31] >> [BIB](https://doi.org/10.1093/bib/bbae162)


### 1.3 Structure

**Modelling peptideâ€“protein complexes: docking, simulations and machine learning**  
Arup Mondal, Liwei Chang and Alberto Perez  
[**2022**-8-26] >> [QRB Discovery](https://doi.org/10.1017/qrd.2022.14)


## 2. Representation

## 3. Proprty Prediction

### 3.1 Traditional

**Beware of extreme calculated lipophilicity when designing cyclic peptides**  
Vasanthanathan Poongavanam, Duc Duy Vo & Jan Kihlberg  
[**2024**-9-19] >> [Nat. Chem. Biol.](https://doi.org/10.1038/s41589-024-01715-0) â€¢ [SI](https://www.nature.com/articles/s41589-024-01715-0#MOESM1) â€¢ [Weixin](https://mp.weixin.qq.com/s/B65rJB1i_xrP8fTfbQ3Taw) â€¢ cyclic/clogP


## 4. Structure Modeling

## 5. Design

### 5.1 Sequence-based

**PepINVENT: Generative peptide design beyond the natural amino acids**  
GÃ¶kÃ§e Geylan, Jon Paul Janet, Alessandro Tibo, Jiazhen He, Atanas Patronov, Mikhail Kabeshov, Florian David, Werngard Czechtizky, Ola Engkvist, Leonardo De Maria  
[**2024**-9-21] >> [Axive](https://doi.org/10.48550/arXiv.2409.14040) â€¢ RL/Molecular AI/AstraZeneca

**Design of Peptide Binders to Conformationally Diverse Targets with Contrastive Language Modeling**  
Suhaas Bhat, Kalyan Palepu, ..., Pranam Chatterjee  
[**2024**-7-22] >> [Arxive](https://doi.org/10.1101/2023.06.26.546591) â€¢ [zenodo](https://zenodo.org/doi/10.5281/zenodo.10971077) â€¢ [huggingface](https://huggingface.co/ubiquitx/pepprclip) â€¢ Pipline
  
<details>
<summary>ğŸ” Abstract</summary>
<p>é’ˆå¯¹éš¾ä»¥æˆè¯çš„è›‹ç™½è´¨è®¾è®¡ç»“åˆå‰‚æ˜¯è¯ç‰©å¼€å‘ä¸­çš„éš¾é¢˜ï¼Œå°¤å…¶æ˜¯æ— åºæˆ–æ„è±¡ä¸ç¨³å®šçš„è›‹ç™½ã€‚æˆ‘ä»¬æå‡ºäº†ä¸€ç§é€šç”¨ç®—æ³•æ¡†æ¶ï¼Œåˆ©ç”¨ç›®æ ‡è›‹ç™½çš„æ°¨åŸºé…¸åºåˆ—è®¾è®¡çŸ­é“¾çº¿æ€§å¤šè‚½ã€‚é€šè¿‡å¯¹ESM-2è›‹ç™½è¯­è¨€æ¨¡å‹çš„æ½œåœ¨ç©ºé—´è¿›è¡Œé«˜æ–¯æ‰°åŠ¨ç”Ÿæˆå¤šè‚½å€™é€‰åºåˆ—ï¼Œå¹¶é€šè¿‡åŸºäºCLIPçš„å¯¹æ¯”å­¦ä¹ æ¶æ„ç­›é€‰é¶å‘é€‰æ‹©æ€§ã€‚æœ€ç»ˆåˆ›å»ºäº†Peptide Prioritization via CLIPï¼ˆPepPrCLIPï¼‰ç®¡é“ï¼Œå¹¶åœ¨å®éªŒä¸­éªŒè¯äº†è¿™äº›å¤šè‚½çš„æœ‰æ•ˆæ€§ï¼Œæ—¢å¯ä½œä¸ºæŠ‘åˆ¶å‰‚ï¼Œä¹Ÿå¯é€šè¿‡ä¸E3æ³›ç´ è¿æ¥é…¶èåˆé™è§£å¤šç§è›‹ç™½é¶æ ‡ã€‚è¯¥ç­–ç•¥æ— éœ€ç¨³å®šçš„ä¸‰çº§ç»“æ„ï¼Œèƒ½å¤Ÿé¶å‘æ— åºå’Œéš¾ä»¥æˆè¯çš„è›‹ç™½è´¨ï¼Œå¦‚è½¬å½•å› å­å’Œèåˆè‡´ç™Œè›‹ç™½ã€‚</p>
</details>  

**HELM-GPT: de novo macrocyclic peptide design using generative pre-trained transformer**  
Xiaopeng Xu,   Chencheng Xu, Wenjia He, Lesong Wei, Haoyang Li, Juexiao Zhou, Ruochi Zhang, Yu Wang, Yuanpeng Xiong, Xin Gao  
[**2024**-6-12] >> [Bioinformatics](https://doi.org/10.1093/bioinformatics/btae364) â€¢ [Github](https://github.com/charlesxu90/helm-gpt) â€¢ GPT/HELM/Macrocyclic/RL

**Design of linear and cyclic peptide binders of different lengths from protein sequence information**  
Qiuzhen Li, Efstathios Nikolaos Vlachos, Patrick Bryant  
[**2024**-10-12] >> [Arxive](https://doi.org/10.1101/2024.06.20.599739) â€¢ [zenodo](https://zenodo.org/uploads/11543503) â€¢ [EvoBind](https://github.com/patrickbryant1/EvoBind) â€¢ cyclic


### 5.2 Structure-based

**Full-Atom Peptide Design with Geometric Latent Diffusion**  
Xiangzhe Kong, Yinjun Jia, Wenbing Huang, Yang Liu  
[**2024**-2-21] >> NeurIPS/[Arxive](https://doi.org/10.48550/arXiv.2402.13555) â€¢ [code](https://github.com/THUNLP-MT/PepGLAD) â€¢ Full-Atom/Diffusion

**Target-Specific De Novo Peptide Binder Design with DiffPepBuilder**  
Fanhao Wang, Yuzhe Wang, Laiyi Feng, Changsheng Zhang, and Luhua Lai  
[**2024**-9-4] >> [JCIM](https://doi.org/10.1021/acs.jcim.4c00975) â€¢ high â€¢ [GitHub](https://github.com/YuzheWangPKU/DiffPepBuilder) â€¢ Diffusion/Luhua Lai/AfDeisgn/ProteinMPNN/MD
  
<details>
<summary>ğŸ” Abstract</summary>
<p>Despite the exciting progress in target-specific de novo protein binder design, peptide binder design remains challenging due to the flexibility of peptide structures and the scarcity of protein-peptide complex structure data. In this study, we curated a large synthetic data set, referred to as PepPC-F, from the abundant proteinâˆ’protein interface data and developed DiffPepBuilder, a de novo target-specific peptide binder generation method that utilizes an SE(3)-equivariant diffusion model trained on PepPC-F to codesign peptide sequences and structures. DiffPepBuilder also introduces disulfide bonds to stabilize the generated peptide structures. We tested DiffPepBuilder on 30 experimentally verified strong peptide binders with available proteinâˆ’peptide complex structures. DiffPepBuilder was able to effectively recall the native structures and sequences of the peptide ligands and to generate novel peptide binders with improved binding free energy. We subsequently conducted de novo generation case studies on three targets. In both the regeneration test and case studies, DiffPepBuilder outperformed AfDesign and RFdiffusion coupled with ProteinMPNN, in terms of sequence and structure recall, interface quality, and structural diversity. Molecular dynamics simulations confirmed that the introduction of disulfide bonds enhanced the structural rigidity and binding performance of the generated peptides. As a general peptide binder de novo design tool, DiffPepBuilder can be used to design peptide binders for given protein targets with three-dimensional and binding site information.</p>
</details>  

**Denovo design of modular peptide-binding proteins by superhelical matching**  
Kejia Wu, Hua Bai, ..., Emmanuel Derivery, Daniel Adriano Silva, David Baker  
[**2023**-3-5] >> [Nature](https://doi.org/10.1038/s41586-023-05909-9) â€¢ high â€¢ [data](https://files.ipd.uw.edu/pub/2023_modular_peptide_binding_proteins/all_data_modular_peptide_binding_proteins.tar.gz) â€¢ [GitHub](https://github.com/tjs23/prot_pep_scan) â€¢ David Baker
  
<details>
<summary>ğŸ” Abstract</summary>
<p>General approaches for designing sequence-specific peptide-binding proteins would have wide utility in proteomics and synthetic biology. However, designing peptide-binding proteins is challenging, as most peptides do not have defined structures in isolation, and hydrogen bonds must be made to the buried polar groups in the peptide backbone1â€“3. Here, inspired by natural and re-engineered proteinpeptide systems4â€“11, we set out to design proteins made out of repeating units that bind peptides with repeating sequences, with a one-to-one correspondence between the repeat units of the protein and those of the peptide. We use geometric hashing to identify protein backbones and peptide-docking arrangements that are compatible with bidentate hydrogen bonds between the side chains of the protein and the peptide backbone12. The remainder of the protein sequence is then optimized for folding and peptide binding. We design repeat proteins to bind to six different tripeptide-repeat sequences in polyproline II conformations. The proteins are hyperstable and bind to four to six tandem repeats of their tripeptide targets with nanomolar to picomolar affinities in vitro and in living cells. Crystal structures reveal repeating interactions between protein and peptide interactions as designed, including ladders of hydrogen bonds from protein side chains to peptide backbones. By redesigning the binding interfaces of individual repeat units, specificity can be achieved for non-repeating peptide sequences and for disordered regions of native proteins.</p>
</details>  


## 6. Others

### 6.1 PROTAC

**A Top-Down Design Approach for Generating a Peptide PROTAC Drug Targeting Androgen Receptor for Androgenetic Alopecia Therapy**  
Bohan MaDonghua LiuZhe WangDize ZhangYanlin Jian, et. al.  
[**2021**-6-5] >> [JMC](https://doi.org/10.1021/acs.jmedchem.4c00828) â€¢ [Weixin](https://mp.weixin.qq.com/s/xeJWFVcV5LkIlVJ1Zxf5Eg) â€¢ PROTAC


### 6.2 Protein Binders

**BindCraft: one-shot design of functional protein binders**  
Martin Pacesa, Lennart Nickel, ..., Sergey Ovchinnikov, Bruno E. Correia  
[**2024**-10-1] >> [Arxive](https://doi.org/10.1101/2024.09.30.615802) â€¢ high â€¢ [GitHub](https://github.com/martinpacesa/BindCraft)
  
<details>
<summary>ğŸ” Abstract</summary>
<p>BindCraft is an open-source, automated pipeline for <em>de novo</em> protein binder design, achieving experimental success rates of 10-100%. Using deep learning models like AlphaFold2, BindCraft generates high-affinity binders without the need for high-throughput screening or prior knowledge of binding sites. It has been successfully applied to challenging targets, including cell-surface receptors, allergens, and CRISPR-Cas9. In one example, the binders reduced IgE binding to birch allergens in patient samples, showcasing its potential in therapeutics, diagnostics, and biotechnology.</p>
</details>  


### 6.3 Therapeutic

**Trends in peptide drug discovery**  
Markus Muttenthaler, GlennF.King, DavidJ.Adams and Paul F. Alewood  
[**2021**-04-01] >> [Nature Reviews Drug Discovery](https://doi.org/10.1038/s41573-020-00135-8) â€¢ high


## Cntribution

[Contributions](https://github.com/zhaisilong/awesome-peptide/blob/main/CONTRIBUTING.md) and [suggestions](https://github.com/zhaisilong/awesome-peptide/issues) are warmly welcome! Community Values, Guiding Principles, and Commitments for the Responsible Development of AI for Peptide Design

## See Also

- [List of papers about Protein Design using Deep Learning](https://github.com/Peldom/papers_for_protein_design_using_DL)
- [Machine learning for proteins](https://github.com/yangkky/Machine-learning-for-proteins)
